docker/Dockerfile.pdo-tp-ccf

```diff
@@ -50,7 +50,7 @@ COPY . /tmp/build-src
 # get the pdo tp source code
 RUN mkdir -p /pdo/dev
 RUN cd /pdo \
-	&& git clone --single-branch --branch ${PDO_REPO_BRANCH} --recurse-submodules ${PDO_REPO_URL} private-data-objects
+	&& git clone --single-branch --branch ${PDO_REPO_BRANCH} ${PDO_REPO_URL} private-data-objects
 WORKDIR /pdo/
```

Comment:
this seems to be a problem. why are you dropping the recurse here? in particular, this has nothing to do with sawtooth and it might be necessary for submodules in wamr.

Reply:
Done

Reply:
regarding the recurse... it may not be necessary but anything that introduces unnecessary differences between the containers will ultimately translate into problems. we've been talking about building the enclave in the CCF container just so we get the right MRENCLAVE to register. this would break that attempt.